services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/oauth/NIHOAuth2ProviderTest.java

```diff
@@ -0,0 +1,59 @@
+package org.sagebionetworks.repo.manager.oauth;
+
+import org.junit.Before;
```

Comment:
We have an agreement in the Synapse team that whenever one of us 'touches' legacy code that uses JUnit 4, they will upgrade that test class to JUnit 5.